com.unity.cinemachine/Editor/Overlays/CinemachineToolbarOverlay.cs

```diff
@@ -144,11 +144,11 @@ class TrackedObjectOffsetTool : CinemachineTool
     /// By default, CinemachineToolSettingsOverlay.customToolbarItems is null.
     /// </summary>
     [Overlay(typeof(SceneView), "Cinemachine Tool Settings")]
-    [Icon("Packages/com.unity.cinemachine/Gizmos/cm_logo.png")]
+    [Icon("Packages/com.unity.cinemachine//Editor/EditorResources/Icons/CmCamera/Dark/VirtualCamera@256.png")]
```

Comment:
How can I see this icon in the editor?  When is it displayed?
Also: this toolbar appears to be empty.  Why do we have it?

Reply:
This is used with Freelook only to select rigs in the scene view.
In 2022.2, I don't know how to open the overlay inspector, but in other versions you can open it with "space" key.